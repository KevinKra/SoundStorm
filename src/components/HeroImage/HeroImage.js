import React from "react";
import "./HeroImage.scss";

export default function HeroImage(props) {
  const styles = {
    width: `${props.width}`,
    height: `${props.height}`,
    backgroundImage: `url(${props.image})`,
    backgroundSize: "cover",
    backgroundPosition: "center"
  };

  return (
    <div className="HeroImage">
      <div style={styles} />
    </div>
  );
}
