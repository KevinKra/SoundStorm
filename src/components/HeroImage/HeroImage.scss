@import "../../_styles/variables.scss";

.HeroImage {
  border-bottom: 1px solid $accent;
  box-shadow: 1px 3px 5px $primary-dark-color;
}
