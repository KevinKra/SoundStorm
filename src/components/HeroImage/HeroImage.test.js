import React from "react";
import HeroImage from "./HeroImage";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("<HeroImage />", () => {
  let wrapper = shallow(<HeroImage />);
  it("should match the snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
