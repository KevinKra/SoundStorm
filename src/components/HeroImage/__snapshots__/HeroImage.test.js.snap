// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeroImage /> should match the snapshot 1`] = `
<div
  className="HeroImage"
>
  <div
    style={
      Object {
        "backgroundImage": "url(undefined)",
        "backgroundPosition": "center",
        "backgroundSize": "cover",
        "height": "undefined",
        "width": "undefined",
      }
    }
  />
</div>
`;
